src/setuptools_scm/scm_workdir.py

```diff
@@ -0,0 +1,13 @@
+from .utils import do_ex, do, require_command
+
+
+class Workdir:
+    def __init__(self, path):
+        require_command(self.COMMAND)
```

Comment:
It seems that the command is used only at a few specified places,

If the requirement is checked in the named constructors, it may not be necessary to explicitly handle certain details